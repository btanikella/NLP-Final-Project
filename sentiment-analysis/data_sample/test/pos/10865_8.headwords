started
gave
thought
get
say
