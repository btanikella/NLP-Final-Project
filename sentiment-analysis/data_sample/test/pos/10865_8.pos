PRP VBD VBG DT VBG DT JJS  FW VBD JJ TO VB IN DT NN VBD RP TO VB JJ  RB VBG IN NNS  IN WRB PRP DT NNS VBD NN
PRP VBD PRP DT NN TO VB NNP NNP IN NN IN DT JJ NN  PRP VBZ DT JJR NN IN FW VBN  IN NNS PRP VBD DT NN IN IN NN
PRP VBD JJ NN VBZ PRP$ NNP NN NNS VBP  IN PRP VBZ JJ TO RBS JJ JJ NNS  PRP VBZ IN DT FW JJ NNS WRB VBN IN NN NN  PRP MD VB JJ CC WRB PRP RB NNS IN WP CC WP PRP RB VBZ PRP RB VBZ DT NN DT NN RBR JJ
RB PRP VBP IN DT NN PRP MD VB PRP VBZ PRP VBN TO PRP  IN IN PRP VBP CD NN RB DT NN IN DT NN NN VBP NN  IN NN FW VBD DT NN IN DT NN CC FW VB TO NN WDT VBZ DT IN PRP VBP RB VB DT IN PRP
RB FW MD VB IN DT NN VBD JJ VBG CC DT JJ NN IN PRP  IN FW VBD PRP
