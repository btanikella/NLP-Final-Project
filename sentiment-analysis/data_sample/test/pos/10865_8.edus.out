Sentence #1 (35 tokens):
I started watching this expecting the worst, i was happy to find that the film turned out to be enjoyable, slightly confusing in parts, like when they all justs started singing.
[Text=I CharacterOffsetBegin=4 CharacterOffsetEnd=5 PartOfSpeech=PRP] [Text=started CharacterOffsetBegin=6 CharacterOffsetEnd=13 PartOfSpeech=VBD] [Text=watching CharacterOffsetBegin=14 CharacterOffsetEnd=22 PartOfSpeech=VBG] [Text=this CharacterOffsetBegin=23 CharacterOffsetEnd=27 PartOfSpeech=DT] [Text=expecting CharacterOffsetBegin=28 CharacterOffsetEnd=37 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=38 CharacterOffsetEnd=41 PartOfSpeech=DT] [Text=worst CharacterOffsetBegin=42 CharacterOffsetEnd=47 PartOfSpeech=JJS] [Text=, CharacterOffsetBegin=47 CharacterOffsetEnd=48 PartOfSpeech=,] [Text=i CharacterOffsetBegin=49 CharacterOffsetEnd=50 PartOfSpeech=FW] [Text=was CharacterOffsetBegin=51 CharacterOffsetEnd=54 PartOfSpeech=VBD] [Text=happy CharacterOffsetBegin=55 CharacterOffsetEnd=60 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=61 CharacterOffsetEnd=63 PartOfSpeech=TO] [Text=find CharacterOffsetBegin=64 CharacterOffsetEnd=68 PartOfSpeech=VB] [Text=that CharacterOffsetBegin=69 CharacterOffsetEnd=73 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=74 CharacterOffsetEnd=77 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=78 CharacterOffsetEnd=82 PartOfSpeech=NN] [Text=turned CharacterOffsetBegin=83 CharacterOffsetEnd=89 PartOfSpeech=VBD] [Text=out CharacterOffsetBegin=90 CharacterOffsetEnd=93 PartOfSpeech=RP] [Text=to CharacterOffsetBegin=94 CharacterOffsetEnd=96 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=97 CharacterOffsetEnd=99 PartOfSpeech=VB] [Text=enjoyable CharacterOffsetBegin=100 CharacterOffsetEnd=109 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=109 CharacterOffsetEnd=110 PartOfSpeech=,] [Text=slightly CharacterOffsetBegin=111 CharacterOffsetEnd=119 PartOfSpeech=RB] [Text=confusing CharacterOffsetBegin=120 CharacterOffsetEnd=129 PartOfSpeech=VBG] [Text=in CharacterOffsetBegin=130 CharacterOffsetEnd=132 PartOfSpeech=IN] [Text=parts CharacterOffsetBegin=133 CharacterOffsetEnd=138 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=138 CharacterOffsetEnd=139 PartOfSpeech=,] [Text=like CharacterOffsetBegin=140 CharacterOffsetEnd=144 PartOfSpeech=IN] [Text=when CharacterOffsetBegin=145 CharacterOffsetEnd=149 PartOfSpeech=WRB] [Text=they CharacterOffsetBegin=150 CharacterOffsetEnd=154 PartOfSpeech=PRP] [Text=all CharacterOffsetBegin=155 CharacterOffsetEnd=158 PartOfSpeech=DT] [Text=justs CharacterOffsetBegin=159 CharacterOffsetEnd=164 PartOfSpeech=NNS] [Text=started CharacterOffsetBegin=165 CharacterOffsetEnd=172 PartOfSpeech=VBD] [Text=singing CharacterOffsetBegin=173 CharacterOffsetEnd=180 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=180 CharacterOffsetEnd=181 PartOfSpeech=.] 
root(ROOT-0, started-2)
nsubj(started-2, I-1)
xcomp(started-2, watching-3)
nsubj(expecting-5, this-4)
dep(watching-3, expecting-5)
det(worst-7, the-6)
dobj(expecting-5, worst-7)
punct(worst-7, ,-8)
nsubj(happy-11, i-9)
cop(happy-11, was-10)
rcmod(worst-7, happy-11)
aux(find-13, to-12)
xcomp(happy-11, find-13)
mark(turned-17, that-14)
det(film-16, the-15)
nsubj(turned-17, film-16)
ccomp(find-13, turned-17)
prt(turned-17, out-18)
aux(enjoyable-21, to-19)
cop(enjoyable-21, be-20)
xcomp(turned-17, enjoyable-21)
punct(turned-17, ,-22)
advmod(confusing-24, slightly-23)
xcomp(turned-17, confusing-24)
prep(confusing-24, in-25)
pobj(in-25, parts-26)
punct(confusing-24, ,-27)
prep(confusing-24, like-28)
advmod(started-33, when-29)
nsubj(started-33, they-30)
det(justs-32, all-31)
nsubj(started-33, justs-32)
pcomp(like-28, started-33)
dobj(started-33, singing-34)
punct(started-2, .-35)

Sentence #2 (35 tokens):
It gave me a chance to see Daniel Wu in action for the first time, he is a better actor than i thought, at times he seemed a bit out of place.
[Text=It CharacterOffsetBegin=186 CharacterOffsetEnd=188 PartOfSpeech=PRP] [Text=gave CharacterOffsetBegin=189 CharacterOffsetEnd=193 PartOfSpeech=VBD] [Text=me CharacterOffsetBegin=194 CharacterOffsetEnd=196 PartOfSpeech=PRP] [Text=a CharacterOffsetBegin=197 CharacterOffsetEnd=198 PartOfSpeech=DT] [Text=chance CharacterOffsetBegin=199 CharacterOffsetEnd=205 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=206 CharacterOffsetEnd=208 PartOfSpeech=TO] [Text=see CharacterOffsetBegin=209 CharacterOffsetEnd=212 PartOfSpeech=VB] [Text=Daniel CharacterOffsetBegin=213 CharacterOffsetEnd=219 PartOfSpeech=NNP] [Text=Wu CharacterOffsetBegin=220 CharacterOffsetEnd=222 PartOfSpeech=NNP] [Text=in CharacterOffsetBegin=223 CharacterOffsetEnd=225 PartOfSpeech=IN] [Text=action CharacterOffsetBegin=226 CharacterOffsetEnd=232 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=233 CharacterOffsetEnd=236 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=237 CharacterOffsetEnd=240 PartOfSpeech=DT] [Text=first CharacterOffsetBegin=241 CharacterOffsetEnd=246 PartOfSpeech=JJ] [Text=time CharacterOffsetBegin=247 CharacterOffsetEnd=251 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=251 CharacterOffsetEnd=252 PartOfSpeech=,] [Text=he CharacterOffsetBegin=253 CharacterOffsetEnd=255 PartOfSpeech=PRP] [Text=is CharacterOffsetBegin=256 CharacterOffsetEnd=258 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=259 CharacterOffsetEnd=260 PartOfSpeech=DT] [Text=better CharacterOffsetBegin=261 CharacterOffsetEnd=267 PartOfSpeech=JJR] [Text=actor CharacterOffsetBegin=268 CharacterOffsetEnd=273 PartOfSpeech=NN] [Text=than CharacterOffsetBegin=274 CharacterOffsetEnd=278 PartOfSpeech=IN] [Text=i CharacterOffsetBegin=279 CharacterOffsetEnd=280 PartOfSpeech=FW] [Text=thought CharacterOffsetBegin=281 CharacterOffsetEnd=288 PartOfSpeech=VBN] [Text=, CharacterOffsetBegin=288 CharacterOffsetEnd=289 PartOfSpeech=,] [Text=at CharacterOffsetBegin=290 CharacterOffsetEnd=292 PartOfSpeech=IN] [Text=times CharacterOffsetBegin=293 CharacterOffsetEnd=298 PartOfSpeech=NNS] [Text=he CharacterOffsetBegin=299 CharacterOffsetEnd=301 PartOfSpeech=PRP] [Text=seemed CharacterOffsetBegin=302 CharacterOffsetEnd=308 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=309 CharacterOffsetEnd=310 PartOfSpeech=DT] [Text=bit CharacterOffsetBegin=311 CharacterOffsetEnd=314 PartOfSpeech=NN] [Text=out CharacterOffsetBegin=315 CharacterOffsetEnd=318 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=319 CharacterOffsetEnd=321 PartOfSpeech=IN] [Text=place CharacterOffsetBegin=322 CharacterOffsetEnd=327 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=327 CharacterOffsetEnd=328 PartOfSpeech=.] 
root(ROOT-0, gave-2)
nsubj(gave-2, It-1)
iobj(gave-2, me-3)
det(chance-5, a-4)
dobj(gave-2, chance-5)
aux(see-7, to-6)
infmod(chance-5, see-7)
nn(Wu-9, Daniel-8)
dobj(see-7, Wu-9)
prep(see-7, in-10)
pobj(in-10, action-11)
prep(action-11, for-12)
det(time-15, the-13)
amod(time-15, first-14)
pobj(for-12, time-15)
punct(gave-2, ,-16)
nsubj(actor-21, he-17)
cop(actor-21, is-18)
det(actor-21, a-19)
amod(actor-21, better-20)
dep(gave-2, actor-21)
mark(bit-31, than-22)
dep(bit-31, i-23)
dep(bit-31, thought-24)
punct(bit-31, ,-25)
prep(bit-31, at-26)
pobj(at-26, times-27)
nsubj(bit-31, he-28)
cop(bit-31, seemed-29)
det(bit-31, a-30)
dep(actor-21, bit-31)
prep(bit-31, out-32)
pcomp(out-32, of-33)
pobj(of-33, place-34)
punct(gave-2, .-35)

Sentence #3 (60 tokens):
I thought purple storm deserves its Hong kong legends release, as it is different to most other HK films, it is about a mans emotional struggles when confronted with memory loss, it may sound corny but when he eventually pieces out what and who he actually is it really makes the film a lot more interesting.
[Text=I CharacterOffsetBegin=333 CharacterOffsetEnd=334 PartOfSpeech=PRP] [Text=thought CharacterOffsetBegin=335 CharacterOffsetEnd=342 PartOfSpeech=VBD] [Text=purple CharacterOffsetBegin=343 CharacterOffsetEnd=349 PartOfSpeech=JJ] [Text=storm CharacterOffsetBegin=350 CharacterOffsetEnd=355 PartOfSpeech=NN] [Text=deserves CharacterOffsetBegin=356 CharacterOffsetEnd=364 PartOfSpeech=VBZ] [Text=its CharacterOffsetBegin=365 CharacterOffsetEnd=368 PartOfSpeech=PRP$] [Text=Hong CharacterOffsetBegin=369 CharacterOffsetEnd=373 PartOfSpeech=NNP] [Text=kong CharacterOffsetBegin=374 CharacterOffsetEnd=378 PartOfSpeech=NN] [Text=legends CharacterOffsetBegin=379 CharacterOffsetEnd=386 PartOfSpeech=NNS] [Text=release CharacterOffsetBegin=387 CharacterOffsetEnd=394 PartOfSpeech=VBP] [Text=, CharacterOffsetBegin=394 CharacterOffsetEnd=395 PartOfSpeech=,] [Text=as CharacterOffsetBegin=396 CharacterOffsetEnd=398 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=399 CharacterOffsetEnd=401 PartOfSpeech=PRP] [Text=is CharacterOffsetBegin=402 CharacterOffsetEnd=404 PartOfSpeech=VBZ] [Text=different CharacterOffsetBegin=405 CharacterOffsetEnd=414 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=415 CharacterOffsetEnd=417 PartOfSpeech=TO] [Text=most CharacterOffsetBegin=418 CharacterOffsetEnd=422 PartOfSpeech=RBS] [Text=other CharacterOffsetBegin=423 CharacterOffsetEnd=428 PartOfSpeech=JJ] [Text=HK CharacterOffsetBegin=429 CharacterOffsetEnd=431 PartOfSpeech=JJ] [Text=films CharacterOffsetBegin=432 CharacterOffsetEnd=437 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=437 CharacterOffsetEnd=438 PartOfSpeech=,] [Text=it CharacterOffsetBegin=439 CharacterOffsetEnd=441 PartOfSpeech=PRP] [Text=is CharacterOffsetBegin=442 CharacterOffsetEnd=444 PartOfSpeech=VBZ] [Text=about CharacterOffsetBegin=445 CharacterOffsetEnd=450 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=451 CharacterOffsetEnd=452 PartOfSpeech=DT] [Text=mans CharacterOffsetBegin=453 CharacterOffsetEnd=457 PartOfSpeech=FW] [Text=emotional CharacterOffsetBegin=458 CharacterOffsetEnd=467 PartOfSpeech=JJ] [Text=struggles CharacterOffsetBegin=468 CharacterOffsetEnd=477 PartOfSpeech=NNS] [Text=when CharacterOffsetBegin=478 CharacterOffsetEnd=482 PartOfSpeech=WRB] [Text=confronted CharacterOffsetBegin=483 CharacterOffsetEnd=493 PartOfSpeech=VBN] [Text=with CharacterOffsetBegin=494 CharacterOffsetEnd=498 PartOfSpeech=IN] [Text=memory CharacterOffsetBegin=499 CharacterOffsetEnd=505 PartOfSpeech=NN] [Text=loss CharacterOffsetBegin=506 CharacterOffsetEnd=510 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=510 CharacterOffsetEnd=511 PartOfSpeech=,] [Text=it CharacterOffsetBegin=512 CharacterOffsetEnd=514 PartOfSpeech=PRP] [Text=may CharacterOffsetBegin=515 CharacterOffsetEnd=518 PartOfSpeech=MD] [Text=sound CharacterOffsetBegin=519 CharacterOffsetEnd=524 PartOfSpeech=VB] [Text=corny CharacterOffsetBegin=525 CharacterOffsetEnd=530 PartOfSpeech=JJ] [Text=but CharacterOffsetBegin=531 CharacterOffsetEnd=534 PartOfSpeech=CC] [Text=when CharacterOffsetBegin=535 CharacterOffsetEnd=539 PartOfSpeech=WRB] [Text=he CharacterOffsetBegin=540 CharacterOffsetEnd=542 PartOfSpeech=PRP] [Text=eventually CharacterOffsetBegin=543 CharacterOffsetEnd=553 PartOfSpeech=RB] [Text=pieces CharacterOffsetBegin=554 CharacterOffsetEnd=560 PartOfSpeech=NNS] [Text=out CharacterOffsetBegin=561 CharacterOffsetEnd=564 PartOfSpeech=IN] [Text=what CharacterOffsetBegin=565 CharacterOffsetEnd=569 PartOfSpeech=WP] [Text=and CharacterOffsetBegin=570 CharacterOffsetEnd=573 PartOfSpeech=CC] [Text=who CharacterOffsetBegin=574 CharacterOffsetEnd=577 PartOfSpeech=WP] [Text=he CharacterOffsetBegin=578 CharacterOffsetEnd=580 PartOfSpeech=PRP] [Text=actually CharacterOffsetBegin=581 CharacterOffsetEnd=589 PartOfSpeech=RB] [Text=is CharacterOffsetBegin=590 CharacterOffsetEnd=592 PartOfSpeech=VBZ] [Text=it CharacterOffsetBegin=593 CharacterOffsetEnd=595 PartOfSpeech=PRP] [Text=really CharacterOffsetBegin=596 CharacterOffsetEnd=602 PartOfSpeech=RB] [Text=makes CharacterOffsetBegin=603 CharacterOffsetEnd=608 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=609 CharacterOffsetEnd=612 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=613 CharacterOffsetEnd=617 PartOfSpeech=NN] [Text=a CharacterOffsetBegin=618 CharacterOffsetEnd=619 PartOfSpeech=DT] [Text=lot CharacterOffsetBegin=620 CharacterOffsetEnd=623 PartOfSpeech=NN] [Text=more CharacterOffsetBegin=624 CharacterOffsetEnd=628 PartOfSpeech=RBR] [Text=interesting CharacterOffsetBegin=629 CharacterOffsetEnd=640 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=640 CharacterOffsetEnd=641 PartOfSpeech=.] 
root(ROOT-0, thought-2)
nsubj(thought-2, I-1)
amod(storm-4, purple-3)
nsubj(deserves-5, storm-4)
ccomp(thought-2, deserves-5)
poss(legends-9, its-6)
nn(legends-9, Hong-7)
nn(legends-9, kong-8)
nsubj(release-10, legends-9)
ccomp(deserves-5, release-10)
punct(release-10, ,-11)
mark(different-15, as-12)
nsubj(different-15, it-13)
cop(different-15, is-14)
advcl(sound-37, different-15)
prep(different-15, to-16)
advmod(films-20, most-17)
amod(films-20, other-18)
amod(films-20, HK-19)
pobj(to-16, films-20)
punct(sound-37, ,-21)
nsubj(sound-37, it-22)
aux(sound-37, is-23)
prep(is-23, about-24)
det(struggles-28, a-25)
amod(struggles-28, mans-26)
amod(struggles-28, emotional-27)
pobj(about-24, struggles-28)
advmod(confronted-30, when-29)
rcmod(struggles-28, confronted-30)
prep(confronted-30, with-31)
nn(loss-33, memory-32)
pobj(with-31, loss-33)
punct(sound-37, ,-34)
nsubj(sound-37, it-35)
aux(sound-37, may-36)
ccomp(release-10, sound-37)
acomp(sound-37, corny-38)
cc(corny-38, but-39)
advmod(pieces-43, when-40)
nsubj(pieces-43, he-41)
advmod(pieces-43, eventually-42)
conj(corny-38, pieces-43)
prt(pieces-43, out-44)
dep(pieces-43, what-45)
cc(what-45, and-46)
dobj(makes-53, who-47)
nsubj(makes-53, he-48)
advmod(makes-53, actually-49)
aux(makes-53, is-50)
nsubj(makes-53, it-51)
advmod(makes-53, really-52)
conj(what-45, makes-53)
det(film-55, the-54)
dobj(makes-53, film-55)
det(lot-57, a-56)
npadvmod(film-55, lot-57)
advmod(interesting-59, more-58)
amod(lot-57, interesting-59)
punct(thought-2, .-60)

Sentence #4 (58 tokens):
Once you get into the film you will find it keeps you gripped to it, as if you miss one bit then a lot of the film wil make sense, for example i missed a bit at the start and i recommend to anyone that watches this that they do not miss any of it.
[Text=Once CharacterOffsetBegin=646 CharacterOffsetEnd=650 PartOfSpeech=RB] [Text=you CharacterOffsetBegin=651 CharacterOffsetEnd=654 PartOfSpeech=PRP] [Text=get CharacterOffsetBegin=655 CharacterOffsetEnd=658 PartOfSpeech=VBP] [Text=into CharacterOffsetBegin=659 CharacterOffsetEnd=663 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=664 CharacterOffsetEnd=667 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=668 CharacterOffsetEnd=672 PartOfSpeech=NN] [Text=you CharacterOffsetBegin=673 CharacterOffsetEnd=676 PartOfSpeech=PRP] [Text=will CharacterOffsetBegin=677 CharacterOffsetEnd=681 PartOfSpeech=MD] [Text=find CharacterOffsetBegin=682 CharacterOffsetEnd=686 PartOfSpeech=VB] [Text=it CharacterOffsetBegin=687 CharacterOffsetEnd=689 PartOfSpeech=PRP] [Text=keeps CharacterOffsetBegin=690 CharacterOffsetEnd=695 PartOfSpeech=VBZ] [Text=you CharacterOffsetBegin=696 CharacterOffsetEnd=699 PartOfSpeech=PRP] [Text=gripped CharacterOffsetBegin=700 CharacterOffsetEnd=707 PartOfSpeech=VBN] [Text=to CharacterOffsetBegin=708 CharacterOffsetEnd=710 PartOfSpeech=TO] [Text=it CharacterOffsetBegin=711 CharacterOffsetEnd=713 PartOfSpeech=PRP] [Text=, CharacterOffsetBegin=713 CharacterOffsetEnd=714 PartOfSpeech=,] [Text=as CharacterOffsetBegin=715 CharacterOffsetEnd=717 PartOfSpeech=IN] [Text=if CharacterOffsetBegin=718 CharacterOffsetEnd=720 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=721 CharacterOffsetEnd=724 PartOfSpeech=PRP] [Text=miss CharacterOffsetBegin=725 CharacterOffsetEnd=729 PartOfSpeech=VBP] [Text=one CharacterOffsetBegin=730 CharacterOffsetEnd=733 PartOfSpeech=CD] [Text=bit CharacterOffsetBegin=734 CharacterOffsetEnd=737 PartOfSpeech=NN] [Text=then CharacterOffsetBegin=738 CharacterOffsetEnd=742 PartOfSpeech=RB] [Text=a CharacterOffsetBegin=743 CharacterOffsetEnd=744 PartOfSpeech=DT] [Text=lot CharacterOffsetBegin=745 CharacterOffsetEnd=748 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=749 CharacterOffsetEnd=751 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=752 CharacterOffsetEnd=755 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=756 CharacterOffsetEnd=760 PartOfSpeech=NN] [Text=wil CharacterOffsetBegin=761 CharacterOffsetEnd=764 PartOfSpeech=NN] [Text=make CharacterOffsetBegin=765 CharacterOffsetEnd=769 PartOfSpeech=VBP] [Text=sense CharacterOffsetBegin=770 CharacterOffsetEnd=775 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=775 CharacterOffsetEnd=776 PartOfSpeech=,] [Text=for CharacterOffsetBegin=777 CharacterOffsetEnd=780 PartOfSpeech=IN] [Text=example CharacterOffsetBegin=781 CharacterOffsetEnd=788 PartOfSpeech=NN] [Text=i CharacterOffsetBegin=789 CharacterOffsetEnd=790 PartOfSpeech=FW] [Text=missed CharacterOffsetBegin=791 CharacterOffsetEnd=797 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=798 CharacterOffsetEnd=799 PartOfSpeech=DT] [Text=bit CharacterOffsetBegin=800 CharacterOffsetEnd=803 PartOfSpeech=NN] [Text=at CharacterOffsetBegin=804 CharacterOffsetEnd=806 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=807 CharacterOffsetEnd=810 PartOfSpeech=DT] [Text=start CharacterOffsetBegin=811 CharacterOffsetEnd=816 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=817 CharacterOffsetEnd=820 PartOfSpeech=CC] [Text=i CharacterOffsetBegin=821 CharacterOffsetEnd=822 PartOfSpeech=FW] [Text=recommend CharacterOffsetBegin=823 CharacterOffsetEnd=832 PartOfSpeech=VB] [Text=to CharacterOffsetBegin=833 CharacterOffsetEnd=835 PartOfSpeech=TO] [Text=anyone CharacterOffsetBegin=836 CharacterOffsetEnd=842 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=843 CharacterOffsetEnd=847 PartOfSpeech=WDT] [Text=watches CharacterOffsetBegin=848 CharacterOffsetEnd=855 PartOfSpeech=VBZ] [Text=this CharacterOffsetBegin=856 CharacterOffsetEnd=860 PartOfSpeech=DT] [Text=that CharacterOffsetBegin=861 CharacterOffsetEnd=865 PartOfSpeech=IN] [Text=they CharacterOffsetBegin=866 CharacterOffsetEnd=870 PartOfSpeech=PRP] [Text=do CharacterOffsetBegin=871 CharacterOffsetEnd=873 PartOfSpeech=VBP] [Text=not CharacterOffsetBegin=874 CharacterOffsetEnd=877 PartOfSpeech=RB] [Text=miss CharacterOffsetBegin=878 CharacterOffsetEnd=882 PartOfSpeech=VB] [Text=any CharacterOffsetBegin=883 CharacterOffsetEnd=886 PartOfSpeech=DT] [Text=of CharacterOffsetBegin=887 CharacterOffsetEnd=889 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=890 CharacterOffsetEnd=892 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=892 CharacterOffsetEnd=893 PartOfSpeech=.] 
root(ROOT-0, get-3)
advmod(get-3, Once-1)
nsubj(get-3, you-2)
prep(get-3, into-4)
det(film-6, the-5)
pobj(into-4, film-6)
nsubj(find-9, you-7)
aux(find-9, will-8)
ccomp(get-3, find-9)
nsubj(keeps-11, it-10)
ccomp(find-9, keeps-11)
nsubj(gripped-13, you-12)
ccomp(keeps-11, gripped-13)
prep(gripped-13, to-14)
pobj(to-14, it-15)
punct(gripped-13, ,-16)
mark(missed-36, as-17)
mark(miss-20, if-18)
nsubj(miss-20, you-19)
advcl(missed-36, miss-20)
num(bit-22, one-21)
nsubj(make-30, bit-22)
advmod(bit-22, then-23)
det(lot-25, a-24)
dep(bit-22, lot-25)
prep(lot-25, of-26)
det(wil-29, the-27)
nn(wil-29, film-28)
pobj(of-26, wil-29)
ccomp(miss-20, make-30)
dobj(make-30, sense-31)
punct(miss-20, ,-32)
prep(miss-20, for-33)
pobj(for-33, example-34)
nsubj(missed-36, i-35)
advcl(gripped-13, missed-36)
det(bit-38, a-37)
dobj(missed-36, bit-38)
prep(missed-36, at-39)
det(start-41, the-40)
pobj(at-39, start-41)
cc(missed-36, and-42)
nsubj(recommend-44, i-43)
conj(missed-36, recommend-44)
prep(recommend-44, to-45)
pobj(to-45, anyone-46)
nsubj(watches-48, that-47)
rcmod(anyone-46, watches-48)
dobj(watches-48, this-49)
mark(miss-54, that-50)
nsubj(miss-54, they-51)
aux(miss-54, do-52)
neg(miss-54, not-53)
ccomp(watches-48, miss-54)
dobj(miss-54, any-55)
prep(any-55, of-56)
pobj(of-56, it-57)
punct(get-3, .-58)

Sentence #5 (22 tokens):
So i can say that this film was worth watching and a grateful surprise for me, that i enjoyed it.
[Text=So CharacterOffsetBegin=898 CharacterOffsetEnd=900 PartOfSpeech=RB] [Text=i CharacterOffsetBegin=901 CharacterOffsetEnd=902 PartOfSpeech=FW] [Text=can CharacterOffsetBegin=903 CharacterOffsetEnd=906 PartOfSpeech=MD] [Text=say CharacterOffsetBegin=907 CharacterOffsetEnd=910 PartOfSpeech=VB] [Text=that CharacterOffsetBegin=911 CharacterOffsetEnd=915 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=916 CharacterOffsetEnd=920 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=921 CharacterOffsetEnd=925 PartOfSpeech=NN] [Text=was CharacterOffsetBegin=926 CharacterOffsetEnd=929 PartOfSpeech=VBD] [Text=worth CharacterOffsetBegin=930 CharacterOffsetEnd=935 PartOfSpeech=JJ] [Text=watching CharacterOffsetBegin=936 CharacterOffsetEnd=944 PartOfSpeech=VBG] [Text=and CharacterOffsetBegin=945 CharacterOffsetEnd=948 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=949 CharacterOffsetEnd=950 PartOfSpeech=DT] [Text=grateful CharacterOffsetBegin=951 CharacterOffsetEnd=959 PartOfSpeech=JJ] [Text=surprise CharacterOffsetBegin=960 CharacterOffsetEnd=968 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=969 CharacterOffsetEnd=972 PartOfSpeech=IN] [Text=me CharacterOffsetBegin=973 CharacterOffsetEnd=975 PartOfSpeech=PRP] [Text=, CharacterOffsetBegin=975 CharacterOffsetEnd=976 PartOfSpeech=,] [Text=that CharacterOffsetBegin=977 CharacterOffsetEnd=981 PartOfSpeech=IN] [Text=i CharacterOffsetBegin=982 CharacterOffsetEnd=983 PartOfSpeech=FW] [Text=enjoyed CharacterOffsetBegin=984 CharacterOffsetEnd=991 PartOfSpeech=VBD] [Text=it CharacterOffsetBegin=992 CharacterOffsetEnd=994 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=994 CharacterOffsetEnd=995 PartOfSpeech=.] 
root(ROOT-0, say-4)
advmod(say-4, So-1)
nsubj(say-4, i-2)
aux(say-4, can-3)
mark(worth-9, that-5)
det(film-7, this-6)
nsubj(worth-9, film-7)
cop(worth-9, was-8)
ccomp(say-4, worth-9)
dep(worth-9, watching-10)
cc(watching-10, and-11)
det(surprise-14, a-12)
amod(surprise-14, grateful-13)
conj(watching-10, surprise-14)
prep(surprise-14, for-15)
pobj(for-15, me-16)
punct(say-4, ,-17)
mark(enjoyed-20, that-18)
nsubj(enjoyed-20, i-19)
ccomp(say-4, enjoyed-20)
dobj(enjoyed-20, it-21)
punct(say-4, .-22)

